src/styles/main.scss

```diff
@@ -1,3 +1,10 @@
-body {
-  margin: 0;
-}
+@import "./blocks/variables/variables";
+
+@import "./blocks/head-block/icon";
+@import "./blocks/head-block/header";
+@import "./blocks/head-block/nav";
+
+@import "./blocks/card-block/content-card";
+@import "./blocks/card-block/stars";
+
+@import "./blocks/main/body";
```

Comment:
The file for 'body' styling is imported last. It's better to import a CSS reset or normalization and basic elements like 'body' or 'html' first before importing more specific components.